src/components/v5/common/ActionSidebar/partials/forms/PaymentBuilderForm/partials/ClaimDelayField/ClaimDelayField.tsx

```diff
@@ -86,6 +86,10 @@ const ClaimDelayField: FC<ClaimDelayFieldProps> = ({
             formattingOptions,
           );
 
+          if (formattedValue === field.value) {
+            return;
+          }
+
```

Comment:
From my testing, in order to avoid the issue upon initially entering a space, I believe we have to update this condition to 

```
          if (
            (!field.value && !formattedValue) ||
            formattedValue === field.value 
          ) {
            return;
          }
```